packages/dumb/src/models/cbg-position-data.model.ts

```diff
@@ -27,11 +27,18 @@
 
 import { type RangeSegmentSliceType } from './enums/range-segment.enum'
 
-export interface CbgPositionData {
+export interface CbgPositionData<T> {
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** ‏Very nice